### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
     "require": {
         "php": "^7.3|^8.0",
```
Comment (jasonmccreary) [inline composer.json:17]: This should be `^8.0` if you plan to only support Laravel 9.

**Comment by ankurk91** [comment]
Laravel 9 is no longer using swift mailer, but this package relies on it.

https://github.com/therobfonz/laravel-mandrill-driver/blob/master/src/MandrillTransport.php#L56

**Comment by jasonmccreary** [comment]
@ankurk91, are you saying this package will not support Laravel 9, or that there is more work to do?

**Comment by therobfonz** [comment]
@jasonmccreary Yeah, there needs to be an upgrade because the underlying mailer has changed. 

I actually am looking for someone who wants to take care of this repository because I haven't used Mandrill in 2 years. Going to see if we want to move it under the Kirschbaum organization to maintain.

**Comment by jasonmccreary** [comment]
@therobfonz, gotcha. This really just bumps the dependencies. But let me know if I can help run some other automation around the Symfony Mailer. We do have that in the [Laravel 9.x Shift](https://laravelshift.com/upgrade-laravel-8-to-laravel-9).

**Comment by jcanepa** [comment]
@therobfonz @jasonmccreary FWIW I'd appreciate someone upgrading the underlying mailer. Thank you both for your contributions.

**Comment by brandonferens** [comment]
I have submitted PR #28 for the Laravel 9 upgrade

**Pull request closed without merging**